packages/table/src/table.tsx

```diff
@@ -95,7 +95,7 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
      * index. Note that if you want to update these values when the user
      * drag-resizes a column, you may define a callback for `onColumnWidthChanged`.
      */
-    columnWidths?: number[];
+    columnWidths?: Array<number | null | undefined>;
```

Comment:
nothing stops you from writing `== null` even if this type changes to `Array<number | undefined>`. what _would_ stop you from writing that is the [no-null-keyword](https://palantir.github.io/tslint/rules/no-null-keyword/) rule. see [here](https://pl.ntr/null-vs-undefined) for my thoughts on null vs undefined